runtime/shared_common/j9shr.tdf

```diff
@@ -1804,37 +1804,6 @@ TraceException=Trc_SHR_RMI_compareROMClasses_NoCachelet Overhead=1 Level=1 Templ
 TraceEntry=Trc_SHR_INIT_hookSerializeSharedCache_entry Noenv Overhead=1 Level=1 Template="INIT HOOK SERIALIZE CACHE: Entering hookSerializeSharedCache"
 TraceExit=Trc_SHR_INIT_hookSerializeSharedCache_exit Noenv Overhead=1 Level=1 Template="INIT HOOK SERIALIZE CACHE: Exiting hookSerializeSharedCache"
 
-TraceEntry=Trc_SHR_OSC_Vmem_startup_Entry NoEnv Overhead=1 Level=1 Template="SH_OSCachevmem::startup: Entering cache name = %s, control dir = %s, cache size = %zu, numSems = %zd, create flags = 0x%zx, verbose flags = 0x%zx, openFlags = 0x%x"
```

Comment:
It's policy to keep existing tracepoint numbers the same, meaning that older tracepoint can't be deleted. They can be updated with the `Obsolete` keyword.

Reply:
Didn't know that. Reverted.